charmcraft.yaml

```diff
@@ -5,13 +5,23 @@ bases:
   - build-on:
     - name: "ubuntu"
       channel: "20.04"
+      architectures:
+      - amd64
+    run-on:
+    - name: "ubuntu"
+      channel: "20.04"
+      architectures:
+      - amd64
+  - build-on:
+    - name: "ubuntu"
+      channel: "20.04"
+      architectures:
+      - arm64
```

Comment:
why is the split necessary?

Reply:
I split it because I don't think the charm built on amd64 can be run on arm64.